widget/playground/src/containers/StyleLayout/StyleLayout.CustomColors.tsx

```diff
@@ -0,0 +1,194 @@
+            trigger={
+              <Row>
+                <Row>
+                  <Colors
+                    mainColor={getMainColor(widgetColor.key, tab, theme)}
+                    secondColor={
+                      isAutoTab
+                        ? getMainColor(widgetColor.key, tab, theme, 'dark') ||
+                          ''
+                        : undefined
+                    }
+                  />
+                  <Divider direction="horizontal" size={16} />
+                  <Typography size="small" variant="body">
+                    {widgetColor.label}
+                  </Typography>
+                </Row>
+                {openCustomColor === widgetColor.key ? (
+                  <ChevronUpIcon size={10} color="gray" />
+                ) : (
+                  <ChevronDownIcon size={10} color="gray" />
+                )}
+              </Row>
+            }>
+            <Divider size={16} />
+
+            {isAutoTab ? (
+              <>
+                <ColorItemPicker
+                  label="Light"
+                  placeholder={widgetColor.label}
+                  color={getMainColor(widgetColor.key, tab, theme, 'light')}
+                  onChangeColor={(color) =>
+                    onChangeColors(widgetColor.key, 'light', color)
+                  }
+                  onReset={() => onResetColor(widgetColor.key, 'light')}
+                  resetDisable={
+                    (!!selectedPresets &&
+                      getMainColor(widgetColor.key, tab, theme, 'light') ===
+                        selectedPresets.light[widgetColor.key]) ||
+                    !selectedPresets
+                  }
+                />
+                <Divider size={16} />
+                <ColorItemPicker
+                  label="Dark"
+                  placeholder={widgetColor.label}
+                  color={getMainColor(widgetColor.key, tab, theme, 'dark')}
+                  onChangeColor={(color) =>
+                    onChangeColors(widgetColor.key, 'dark', color)
+                  }
+                  resetDisable={
+                    (!!selectedPresets &&
+                      getMainColor(widgetColor.key, tab, theme, 'dark') ===
+                        selectedPresets.dark[widgetColor.key]) ||
+                    !selectedPresets
+                  }
+                  onReset={() => onResetColor(widgetColor.key, 'dark')}
+                />
+              </>
+            ) : (
+              <ColorItemPicker
+                label="Main"
+                placeholder={widgetColor.label}
+                color={getMainColor(widgetColor.key, tab, theme) || ''}
+                onChangeColor={(color) =>
+                  onChangeColors(widgetColor.key, tab, color)
+                }
+                onReset={() => onResetColor(widgetColor.key, tab)}
+                resetDisable={
+                  (!!selectedPresets &&
+                    getMainColor(widgetColor.key, tab, theme) ===
```

Comment:
I get a type error in this function evereywhere.

Reply:
fixed